### DISCUSSION TITLE
Ability to create custom themes

### DISCUSSION THREAD

**Discussion by armanossiloko** [grafana/grafana#72053] [General]
I know this has been debated before in #10495 and #4909. Also, I do know that sometimes when people ask about this (e.g #32784), the [question of custom theming](https://github.com/grafana/grafana/issues/32784#issuecomment-823849870) is even ignored. 

I wonder what the reasoning behind declining such a feature request is? While I do understand that creating themes for Grafana and adding them to the default build of Grafana wouldn't make much more sense due to the amount of work that would be required to maintain those, the idea behind a custom theme builder makes sense.

If you were to copy one of the two existing themes, rename them to whatever you liked and then adjust them to your own liking, it would mean that end customers / users would be responsible for keeping their themes up-to-date and would not require anyone here to go and adjust the SCSS or CSS files to fix "something looking odd".

There seems to be a very nice benefit of introducing something like a custom theme builder or the ability to define custom themes, while not putting a huge constraint on anyone maintaining this repository and Grafana altogether.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). As you pointed out, there is already ongoing discussions about this elsewhere. I will be closing this discussion, and it can continue in https://github.com/grafana/grafana/issues/10495
